### DISCUSSION TITLE
gone XCH Block Reward

### DISCUSSION THREAD

**Discussion by ermak-74** [Chia-Network/chia-blockchain#5548] [Support]
during the last two days, the chia version 1.1.5 blockchain program did not work stably. After reinstalling, the reward for the XCH block was lost in the amount of 2. Please help me restore it?

**Comment by lvcivs** [comment]
Is your wallet properly synced?

**Reply by ermak-74** [reply]
please tell me how to sync your wallet correctly.

**Reply by ermak-74** [reply]
the synchronization process is currently underway

**Comment by ermak-74** [comment]
thanks. synced and everything is ok.

**Comment by BrandtH22** [comment]
@ermak-74 , based on your previous message it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
